galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zosmf.manager/src/main/java/dev/galasa/zosmf/IZosmf.java

```diff
@@ -142,10 +146,11 @@ public String toString() {
      * 
      * @param path identifies the zOSMF REST API
      * @param validStatusCodes a {@link List} of acceptable HTTP Status codes. Default to {@link HttpStatus.HttpStatus.SC_OK} when null
+     * @param convertis a data conversion required
```

Comment:
convertis ? :-)

Reply:
Maybe  "@param convert is a data conversion required. If true, data will be converted betwen EBCDIC to ISO8859-1. If false, no data conversion will take place."